### PR TITLE
NXP-28068: fix default-search buttons position

### PR DESCRIPTION
**Pull request by semisse**

### CODE REVIEW

**Comment by semisse** [comment]
Pipeline ok: https://qa.nuxeo.org/jenkins/job/web-ui-pipeline-sfialho/100/

File: elements/search/nuxeo-search-form.js [hunk 1]
```diff
@@ -706,10 +702,23 @@ Polymer({
     return this.selectedSearchIdx > 0;
   },
 
+  _reset() {
```
Comment (nmpcunha) [inline elements/search/nuxeo-search-form.js]: nit: usually we don't change method's location unless we have a strong reason to. It makes the diff more "messy"

File: elements/search/nuxeo-search-form.js [hunk 2]
```diff
@@ -706,10 +702,23 @@ Polymer({
...
+        this.form.searchTerm = this.searchTerm;
+        this.dirty = false;
+      });
+    }
+  },
+
   _clear() {
     this.searchTerm = '';
-    this.isSavedSearch = false;
-    this.selectedSearch = null;
```
Comment (guirenard) [inline elements/search/nuxeo-search-form.js]: Are 100% sure these 2 lines ca be removed?

Comment (semisse) [reply elements/search/nuxeo-search-form.js]: Actually, after giving a closer look, I'm sure we can't remove them. 
The way I was approaching this was something along these lines: this function was running in both default-search and saved search. With this new solution, it only runs on the first one. This way we didn't need to reset the those values. Although, there is a situation when we go back and forward between default and save searches that we actually need to reset them.

**Pull request merged**